### PR TITLE
graphviz 5.0.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/graphviz.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@ class Graphviz < Formula
   on_linux do
     depends_on "byacc" => :build
     depends_on "ghostscript" => :build
+    depends_on "python3" => :build
```
Comment (carlocab) [inline Formula/graphviz.rb:36]: Move this out of the `on_linux` block and do
```ruby
uses_from_macos "python" => :build
```
to fix the Linux audit failure.

**Comment by Porkepix** [comment]
Seems to fail picking up a `11` runner, needs a rerun on it.

**Comment by carlocab** [comment]
CC @Bo98

**Comment by Porkepix** [comment]
The test was green on the previous run, wasn't it…?

**Comment by carlocab** [comment]
```
  curl: (56) LibreSSL SSL_read: error:02FFF036:system library:func(4095):Connection reset by peer, errno 54
  Error: anjuta: Failed to download resource "libunistring"
```
Just a network error.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**